### DISCUSSION TITLE
DeepMD training model with Single Point energy calculations

### DISCUSSION THREAD

**Discussion by stfalxndria** [deepmodeling/deepmd-kit#3416] [Q&A]
Dear DeepMD community,
        I am currently a first year PhD student who is extremely new to machine learning molecular dynamics.
	I am trying to train DeepMD model with separate single point energy calculations (around 2000 different configurations), however, I was unable to find any tutorials on how to do so, most of the information online was on how to train the model with AIMD, which would have all the trajectories in one file.
	It would be great if you can give me some sort of guidance to point me in the right direction. Thank you so much for your patience and help in advance!

Kindest Regards,
Stefani

**Comment by njzjz** [comment]
Hi,

For a low-level solution, you can generate data by yourself. See https://docs.deepmodeling.com/projects/deepmd/en/master/data/data-conv.html for the format.
For a high-level solution, you can use the `dpdata` package. See https://docs.deepmodeling.com/projects/dpdata/.

**Comment by stfalxndria** [comment]
Hi,
thank you so much for the reply! I have used the second method you mentioned and it worked for me!
however, now i am trying to train my model with data obtained from CP2K AIMD and XTB MD simulations and this method has failed.

I have tried both:
dpdata.LabeledSystem(filename, fmt='cp2kdata/e_f') 
and 
dpdata.LabeledSystem(filename, fmt='cp2k/output')

both of which has given me this error:
‘Cp2kOutput' object has no attribute 'energies_list'

I am not sure where to move on from this and what to do next since 

dpdata.LabeledSystem(filename, fmt='cp2kdata/md') and fmt = 'cp2k/aimd_output' has also given me another type of error.

Thanks a lot again!

**Reply by njzjz** [reply]
FYI @robinzyb

**Reply by robinzyb** [reply]
Hello, this means that dpdata didn't find the energy in the output. Could I have your output?

**Reply by stfalxndria** [reply]
Hi, 
thanks for the reply! I was not able to upload it here since the file is massive so here's a link, this is the output of the AIMD simulation.
https://liveuclac-my.sharepoint.com/:f:/g/personal/uccaset_ucl_ac_uk/EjyPhoow0o5ClnEfQUj4xCwBm5Q69T6SLYFpo1o3_YJdMQ?e=fsGBmN

thank you so much!
